arviz/data/inference_data.py

```diff
@@ -298,6 +319,29 @@ def to_dict(self, groups=None, filter_groups=None):
         ret["attrs"] = attrs
         return ret
 
+    def to_json(self, filename, **kwargs):
+        """Write InferenceData to a json file.
+
+        Parameters
+        ----------
+        filename : str
+            Location to write to
+        kwargs : dict
+            kwargs passed to json.dumps()
+
+        Returns
+        -------
+        str
+            Location of json file
+        """
+        idata_dict = self.to_dict()
+        serialised_data = json.dumps(idata_dict, **kwargs)
```

Comment:
Should we just use `.dump`?

This will create a copy of the data in-memory, `dump` might use less ram.